src/docker/build.nim

```diff
@@ -269,10 +269,25 @@ proc launchDockerSubscan(ctx:     DockerInvocation,
   result = runChalkSubScan(usableContexts, "insert").report
   trace("docker: subscan complete.")
 
+proc getRelativePathBetween(base_path: string, path: string) : string =
```

Comment:
lets move to `src/util.nim` as this seems generic enough to be useful other places